docs/style.en-US.md

```diff
@@ -0,0 +1,180 @@
+---
+order: 6
+title: Style
+type: Introduction
+---
+
+This document is mainly about how to use and organize style files in projects. If you want to get a basic knowledge of CSS or look for properties usage, you can refer to the [MDN doc] (https://developer.mozilla.org/en-US/docs/Web/CSS/Reference).
+
+---
+
+## less
+
+Ant Design Pro defaults to using less as the style language, we recommended that you learn about the features of [less](http://lesscss.org/) before using it or sometimes when you have some questions.
+
+## CSS Modules
+
+In the style development process, there are two problems are prominent:
+
+- Global Pollution - CSS file selectors are globally valid. according to the build file generated in the order, the ahead selector's style will be cover by the back one which has the same selector name;
```

Comment:
`CSS selectors are globally valid. Selectors with the same name in different files will be built together, and the former will be covered by the latter.`